### PR TITLE
#535: Clean up README prior to 1.5 release

### LINKED ISSUE #535: Clean up README prior to 1.5 release
It has not been updated in a while.

### PR DESCRIPTION
**Pull request by cwschilly**
Fixes #535

~~Wait to merge until #528 to fix failing tests~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,115 +2,142 @@
...
-
-If the lbaf package is installed you can run LBAF using the following command:
+If the `lbaf` package is installed, you can run LBAF using the following command:
 ```shell
-lbaf -c <config-file-name>
+lbaf -c <config-file-path>
 ```
 
-Or run LBAF from source:
+If you installed dependencies instead, then you will run LBAF from source:
```
Comment (ppebay) [inline README.md:106]: Suggested rephrasing (as previous sentence re: installation in the passive voice)
`If dependencies where installed instead, then run LBAF from source:`

File: README.md [hunk 2]
```diff
@@ -2,115 +2,142 @@
...
+- run all tests defined in `tox.ini`
 - create the `artifacts` directory in main project path
 - create an html coverage report and a pylint report within the `artifacts` directory
 
 ## Usage
 
-### LBAF
-
-If the lbaf package is installed you can run LBAF using the following command:
+If the `lbaf` package is installed, you can run LBAF using the following command:
```
Comment (ppebay) [inline README.md:101]: Consistency in usage of the passive voice:
`If the lbaf package is installed, LBAF can be run using the following command:`

File: README.md [hunk 3]
```diff
@@ -2,115 +2,142 @@
...
 
-If `<config-file-name>` is a relative path then the application will search first from the current working directory, then from the `<project-path>/config` directory.
+### Visualization
 
-A description of each parameter in configuration file can be found at the top of configuration file.
+LBAF can optionally leverage [`vt-tv`](https://github.com/DARMA-tasking/vt-tv), a DARMA-tasking tool built off of [`VTK`](https://vtk.org/), to visualize the work-to-rank mappings, communications, and memory usage of a run.
+
+To get started, you will need to build `VTK` (instructions [here](https://gitlab.kitware.com/vtk/vtk/-/blob/master/Documentation/docs/build_instructions/build.md)).
+
+Then, clone the `vt-tv` repo and install the Python bindings:
```
Comment (ppebay) [inline README.md:127]: repository

File: README.md [hunk 4]
```diff
@@ -126,40 +153,55 @@ python src/lbaf/Applications/LBAF_app.py -c <config-file-name> -v <verbosity-lev
...
 cd <project-path>
 python src/lbaf/imported/JSON_data_files_validator.py
 ```
-This command automatically downloads the JSON_data_files_validator.py script if needed.
 
+_Note: This command automatically downloads the `JSON_data_files_validator.py` script if needed._
+
+#### Usage
+
+These commands assume that LBAF was installed as a package. If you are running from source, you will need to replace the run command as noted above.
```
Comment (ppebay) [inline README.md:201]: `When running from source, replace the run command...`

File: README.md [hunk 5]
```diff
@@ -208,46 +258,37 @@ python src/lbaf/Utils/lbsVTDataExtractor.py
...
+</details>
 
-[//]: # (    ```"python src/Applications/NodeGossiper.py -l /lbaf/data/vt_example_lb_stats/stats -x 4 -y 2 -z 1 -s 0 -f 4 -k 4 -i 4 -c 1 -e"```)
+## LBAF in Literature
 
-[//]: # (  - command to stay inside container, after above command is completed:)
+### [Optimizing Distributed Load Balancing for Workloads with Time-Varying Imbalance](https://ieeexplore.ieee.org/document/9556089)
 
-[//]: # (    ```"/bin/bash"```)
+"This paper explores dynamic load balancing algorithms used by asynchronous many-task (AMT), or ‘taskbased’, programming models to optimize task placement for scientific applications with dynamic workload imbalances."
```
Comment (ppebay) [inline README.md:294]: `task-based' seems better

File: README.md [hunk 6]
```diff
@@ -182,22 +225,29 @@ lbaf-vt-data-files-validator --dir_path=<project-path>/data/nolb-8color-16nodes-
...
 ```
+</details>
 
-## VT data Extractor
+<details>
+<summary><b>VT Data Extractor</b></summary>
 
-VT data Extractor extracts phases from VT stats files.
+<br />
+The VT Data Extractor extracts phases from VT stats files.
```
Comment (ppebay) [inline README.md:234]: I wonder if we should not use small caps for `vt` everywhere now, for consistency.

**Review by ppebay — CHANGES_REQUESTED** [review]
Small changes requested

File: README.md (hunk 1 above)
Comment (cwschilly) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (cwschilly) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (cwschilly) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (cwschilly) [reply README.md]: Fixed

File: README.md (hunk 5 above)
Comment (cwschilly) [reply README.md]: Fixed (I will note that this is taken directly from the paper)

File: README.md (hunk 6 above)
Comment (cwschilly) [reply README.md]: Fixed

**Review by ppebay — APPROVED** [review]
This is a really good clean-up, thank you.

More can always be done but I think we need to draw a line in the sand, it will be perfect for 1.5!

**Pull request merged**